src/devices/CharacterLcd/CharacterLcd.csproj

```diff
@@ -9,17 +9,13 @@
 
   <ItemGroup>
     <Compile Include="Flags.cs" />
-    <Compile Include="GpioControllerAdapter.cs" />
-    <Compile Include="IGpioController.cs" />
     <Compile Include="Hd44780.cs" />
     <Compile Include="Lcd1602.cs" />
     <Compile Include="Lcd2004.cs" />
-    <Compile Include="Mcp23008Adapter.cs" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease*" />
-    <ProjectReference Include="$(MSBuildThisFileDirectory)../Mcp23xxx/Mcp23xxx.csproj" />
+    <ProjectReference Include="..\..\System.Device.Gpio\System.Device.Gpio.csproj" />
```

Comment:
would be nice to setup the local package source perhaps

Reply:
This is just a temporary hack